MOBILE_CLIENT/COMMON/_HIGH/LEAKING_PERSONAL_INFORMATION/description.md

```diff
@@ -1 +1,163 @@
-Personally Identifiable Information (PII) is, according to NIST Special Publication 800-122, a collective term for any information that can be used to distinguish or trace an individual's identity, such as name, social security number, date and place of birth, mother's maiden name, or biometric records; and any other information that is linked or linkable to an individual, such as medical, educational, financial, and employment information.
\ No newline at end of file
+Personally Identifiable Information (PII) is, according to NIST Special Publication 800-122, a collective term for any information that can be used to distinguish or trace an individual's identity, such as name, social security number, date and place of birth, mother's maiden name, or biometric records; and any other information that is linked or linkable to an individual, such as medical, educational, financial, and employment information.
+
+In the context of mobile security, PII leakage occurs when plain text PII information is logged to application logs or a world-readable file making it accessible to all applications on the user device.
+
+=== Kotlin
```

Comment:
```suggestion
=== "Kotlin"
```
Put the name between quotes so that it's rendered as a tab in the docs. Same for the rest.